lti_consumer/lti_xblock.py

```diff
@@ -1235,9 +1235,12 @@ def lti_launch_handler(self, request, suffix=''):  # pylint: disable=unused-argu
             role = LTI_1P1_ROLE_MAP.get(role, 'Student,Learner')
 
             result_sourcedid = self.lis_result_sourcedid
+        # Fails if extract_real_user_data() fails
         except LtiError:
             loader = ResourceLoader(__name__)
-            template = loader.render_django_template('/templates/html/lti_launch_error.html')
+            error_msg = 'Could not get user data for current request'
```

Comment:
this repeats the message we already have on the LtiError object. I think you could just use `except LTIError err:` and than grab `err.message`